data/events.txt

```diff
@@ -2702,16 +2772,36 @@ event "fwc pug defeated"
 		security 0.1
```

Comment:
The security changes to Silver, Rand, and Oblivion are now redundant here since you made the changes when the government flips.

Reply:
I thought about that - I just wasn't sure if there might be some small branch I was overlooking that might skip one or the other. If you say they are redundant, though, I'll remove them.

Reply:
resolved